assets/thunderstore.toml

```diff
@@ -0,0 +1,36 @@
+[config]
+schemaVersion = "0.0.1"
+
+[general]
+repository = "https://thunderstore.io"
+
+[package]
+namespace = ""
```

Comment:
Missing namespace

Reply:
@MaxWasUnavailable What do you want the team name to be for thunderstore?